pyproject.toml

```
@@ -19,7 +19,8 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "google-cloud-bigquery"
 authors = [{ name = "Google LLC", email = "googleapis-packages@google.com" }]
-license = { text = "Apache 2.0" }
+license = { text = "Apache-2.0" }
```

Dicts appear to be deprecated. It appears that the correct format is a simple string:
Fixing this should clear up the failing tests. <fingers crossed>.

```suggestion
license = "Apache-2.0"
```

---

@walid83-0 Take a look. The tests are failing. Here is an example Error:

```
× Getting requirements to build editable did not run successfully.
  │ exit code: 1
  ╰─> [44 lines of output]
      configuration error: `project.license` must be string
      GIVEN VALUE:
          {
              "text": "Apache-2.0"
          }
      
      OFFENDING RULE: 'type'
      
      DEFINITION:
          {
              "type": "string"
          }
      Traceback (most recent call last):
```